tools/scripts/tidy.toml

```diff
@@ -98,4 +98,46 @@ if greater_than ${output_length} 0
         trigger_error ${msg}
     end
 end
-'''
\ No newline at end of file
+'''
+
+[tasks.copy-license]
+description = "Copy the license file to each component for packaging."
+category = "ICU4X Development"
+script_runner = "@duckscript"
+script = '''
+exit_on_error true
+
+glob_pattern = set "./**/Cargo.toml"
+license = canonicalize LICENSE
+
+handle = glob_array ${glob_pattern}
+for path in ${handle}
+    root_dir = dirname ${path}
+    if not is_empty ${root_dir}
+        echo "Copying LICENSE to ${root_dir}/LICENSE"
+        cp ${license} ${root_dir}/LICENSE
+    end
+end
+'''
+
+[tasks.license-file-check]
+description = "Check that each component has a LICENSE file."
+category = "ICU4X Development"
+script_runner = "@duckscript"
+script = '''
+exit_on_error true
+
+glob_pattern = set "./**/Cargo.toml"
+license = canonicalize LICENSE
+
+handle = glob_array ${glob_pattern}
+for path in ${handle}
+    root_dir = dirname ${path}
+    if not is_empty ${root_dir}
+        if not is_path_exists ${root_dir}/LICENSE
+            trigger_error "LICENSE file missing in ${root_dir}"
```

Comment:
suggestion: mention something like "run `cargo make copy-license` to automatically copy the license to all necessary locations"

Reply:
is there any linter on the lower level .toml files themselves? maybe something to enforce consistency (the all have the same `categories` etc) would be useful.

Reply:
Not yet, but that's a good idea!